### PR TITLE
Use error wrappers in all examples

### LINKED ISSUE #2041: Use SnapErrors in examples

### PR DESCRIPTION
**Pull request by FrederikBolding**
Use error wrappers from the SDK in all examples, instead of recommending direct usage of `rpc-errors`.

Fixes #2041

### CODE REVIEW

File: packages/examples/packages/webpack-plugin/webpack.config.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ const config: Configuration = {
     minimize: true,
     minimizer: [
       new TerserPlugin({
-        minify: TerserPlugin.swcMinify,
+        parallel: true,
```
Comment (Mrtenz) [inline packages/examples/packages/webpack-plugin/webpack.config.ts:38]: Why did you change this?

**Review by Mrtenz — APPROVED** [review]

File: packages/examples/packages/webpack-plugin/webpack.config.ts (hunk 1 above)
Comment (FrederikBolding) [reply packages/examples/packages/webpack-plugin/webpack.config.ts:38]: Same reason we can't use the SWC minifier in the CLI. It screws with our post processing of HTML comments
